packages/diagram/src/module/diagrams/content/uml/classifier/sections.ts

```diff
@@ -0,0 +1,50 @@
+import { InterpreterModule, parse } from "@hylimo/core";
+
+/**
+ * Module providing the sections content handler
+ */
+export const sectionsModule = InterpreterModule.create(
+    "uml/classifier/sections",
+    [],
+    [],
+    [
+        ...parse(
+            `
+                scope.internal.sectionsContentHandler = [
+                    {
+                        this.scope = args.scope
+                        scope.sections = list()
+                        scope.section = listWrapper {
+                            sectionIndex = args.section
+                            newSection = it
+                            if(sectionIndex == null) {
+                                scope.sections += newSection
+                            } {
+                                while { scope.sections.length <= sectionIndex } {
+                                    scope.sections += null
```

Comment:
:raised_eyebrow:

Reply:
Also, since you do it below anyway, why not initialize it with `+= list()` directly?

Reply:
`null` has a different meaning than an empty list
Null entries are skipped
E.g. when having an enum with only entries, the compartments 0 and 1 (fields and methods) are `null` and thus not visible.
However if these would be an empty list, there would be 2 very tiny compartments